app/src/main/res/layout-land/fragment_verification_intro.xml

```diff
@@ -17,28 +17,61 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
         >
 
-      <TextView
-          android:id="@+id/title"
-          style="@style/VerifyWallet.Title"
+      <androidx.constraintlayout.widget.ConstraintLayout
+          android:id="@+id/info_container"
           android:layout_width="0dp"
           android:layout_height="wrap_content"
-          android:layout_marginStart="24dp"
-          android:layout_marginTop="27dp"
+          android:layout_marginStart="16dp"
+          android:layout_marginEnd="16dp"
+          android:layout_marginTop="16dp"
+          android:background="@drawable/background_info_card"
+          android:padding="17dp"
           app:layout_constraintEnd_toStartOf="@id/guideline_mid"
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toTopOf="parent"
-          tools:text="@string/card_verification_body_new"
-          />
+          >
+
+        <ImageView
+            android:id="@+id/info_image"
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:src="@drawable/ic_info_new"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+
+        <TextView
+            android:id="@+id/info_description"
+            style="@style/VerifyWallet.Info"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="0dp"
+            android:layout_marginEnd="0dp"
+            android:textColor="@color/styleguide_light_grey"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/info_image"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="@string/verify_card_view_disclaimer"
+            />
+
+      </androidx.constraintlayout.widget.ConstraintLayout>
 
       <RelativeLayout
           android:id="@+id/cc_container"
           android:layout_width="0dp"
           android:layout_height="wrap_content"
-          android:layout_marginTop="@dimen/half_large_margin"
+          android:layout_marginTop="-8dp"
```

Comment:
why to use a negative number?

Reply:
It is necessary to compensate for the margin of the Adyen component, though not ideal